rpm/generic/zfs.spec.in

```diff
@@ -87,6 +87,7 @@ BuildRequires:  libuuid-devel
 BuildRequires:  libblkid-devel
 BuildRequires:  libudev-devel
 BuildRequires:  libattr-devel
+BuildRequires:  openssl-devel
```

Comment:
The core zfs utilities now require `openssl` so we need to add:

```
Requires: openssl
```

I believe dracut will automatically pull in the library dependency for the initramfs.  However, while looking at `./contrib/dracut/90zfs/module-setup.sh.in` a noticed that `installkernel()` has never included the icp module and it should make sure that gets installed.